### PR TITLE
Enable metering proxy for local development

### PR DESCRIPTION
**Pull request by spadgett**
Like #2527, but for metering.

/cc @yaacov @dtaylor113

### CODE REVIEW

File: cmd/bridge/main.go [hunk 1]
```diff
@@ -158,6 +159,12 @@ func main() {
 		offClusterAlertManagerURL.Path = "/api"
 	}
 
+	offClusterMeteringURL := &url.URL{}
+	if *fK8sModeOffClusterMetering != "" && *fK8sMode == "off-cluster" {
```
Comment (yaacov) [inline cmd/bridge/main.go]: in https://github.com/openshift/console/pull/2527 we did ` && *fK8sMode == "off-cluster"` for each off cluster arg:
```
if *fK8sModeOffClusterMetering != "" && *fK8sMode == "off-cluster" {
```
I think it's nicer to move from the code from L149 to L166 into the `case "off-cluster":` in L316 and create `offClusterAlertManagerURL`, `offClusterPrometheusURL` and `offClusterMeteringURL` inside the case, WDYT ?

**Comment by yaacov** [comment]
other then the nitpick above /lgtm

**Comment by spadgett** [comment]
@yaacov thanks, updated

**Comment by spadgett** [comment]
@yaacov ptal

**Pull request merged**